lib/queue.js

```diff
@@ -26,16 +28,9 @@ module.exports = FetchQueue;
 
 FetchQueue.prototype = [];
 FetchQueue.prototype.add = function(protocol, domain, port, path, depth, callback) {
-
-    // For legacy reasons
-    if (depth instanceof Function) {
-        callback = depth;
-        depth = 1;
-    }
-
     depth = depth || 1;
```

Comment:
Oh yeah right, forgot to leave a comment about this! I figured that since the next release target probably is 1.0.0, it might be a good time to clean up any legacy code that's not documented etc. So long as we make sure to make a note of it when releasing